ee/cis/win-10/test/instructions/CIS_2.3.11.5.txt

```diff
@@ -0,0 +1,18 @@
+Expected scenario
+==================
+1) Open "Edit Group Policy" tool and set the following UI path to 'AES128_HMAC_SHA1 AND AES256_HMAC_SHA1 AND Future encryption types':

```

Comment:
Noticed this was a repeat of 2.3.11.4.txt

This one should be: 
Ensure 'Network security: Do not store LAN Manager hash
value on next password change' is set to 'Enabled' (Automated)

Page 263 of https://drive.google.com/file/d/16M2AuKHu_x-WJZWObhl_7MpZkPkKSQaq/view

<3

Reply:
@RachelElysia great catch! I've just fixed this. Thanks!